mirascope/core/mistral/_utils/_setup_call.py

```diff
@@ -15,12 +15,18 @@
     ToolChoice,
 )
 
-from ...base import BaseMessageParam, BaseTool, _utils
+from ...base import BaseCallParams, BaseMessageParam, BaseTool, _utils
 from ..call_params import MistralCallParams
 from ..dynamic_config import MistralDynamicConfig
 from ..tool import MistralTool
 from ._convert_message_params import convert_message_params
 
+_BaseToolT = TypeVar("_BaseToolT", bound=BaseTool)
+
+
+class MistralToolCallParams(BaseCallParams[_BaseToolT]):
+    tool_choice: NotRequired[Literal["required"]]
```

Comment:
this seems like the wrong type? why isn't this `NotRequired[ToolChoice]`?

also see other comment re: `BaseCallKwargs` change

Reply:
Fixed